src/domain/opendatafabric/src/serde/flatbuffers/proxies_generated.rs

```diff
@@ -1464,9 +1465,13 @@ pub struct TransformResponseUnionTableOffset {}
 ////////////////////////////////////////////////////////////////////////////////
 // struct Timestamp, aligned to 4
 #[repr(transparent)]
-#[derive(Clone, Copy, PartialEq, Default)]
+#[derive(Clone, Copy, PartialEq)]
 pub struct Timestamp(pub [u8; 16]);
-
+impl Default for Timestamp {
+    fn default() -> Self {
+        Self([0; 16])
+    }
+}
```

Comment:
A question may arise, why not apply clippy auto-fixes to generated files? 

I've tried doing it this way, but some auto-fixes resulted in compilation errors -- so automating this process (linting the generated files) is not currently proposed